Description: Use GTK look and feel by default.
 Patch rewritten for new code, original patch was from Ubuntu's
 01-default-look-and-feel.dpatch by Mathieu Trudel <mathieu.tl@gmail.com>
Author: David Paleino <dapal@debian.org>
Forwarded: not-needed

--- a/src/org/openstreetmap/josm/tools/PlatformHookUnixoid.java
+++ b/src/org/openstreetmap/josm/tools/PlatformHookUnixoid.java
@@ -99,7 +99,7 @@ public class PlatformHookUnixoid impleme
 
     @Override
     public String getDefaultStyle() {
-        return "javax.swing.plaf.metal.MetalLookAndFeel";
+        return "javax.swing.plaf.gtk.GTKLookAndFeel";
     }
 
     /**
